api/lanes.py

```diff
@@ -1303,30 +1325,19 @@ def __init__(self, library):
         # every collection.
         for collection in sorted(library.collections, key=lambda x: x.name):
             for medium in Edition.FULFILLABLE_MEDIA:
-                for availability in [Facets.AVAILABLE_NOW]:
-                    facets = Facets.default(
-                        library, availability=availability
-                    )
-
-                    # Give each Worklist a name that is distinctive
-                    # and easy for a client to parse.
-                    if collection.data_source:
-                        data_source_name = collection.data_source.name
-                    else:
-                        data_source_name = "[Unknown]"
-                    display_name = "[Collection test] - License source {%s} - Medium {%s} - Availability {%s} - Collection name {%s}" % (data_source_name, medium, availability, collection.name)
-                    child = KnownOverviewFacetsWorkList(facets)
-                    child.initialize(
-                        library, media=[medium], display_name=display_name
-                    )
-                    child.collection_ids = [collection.id]
-                    self.children.append(child)
-
-            # TODO: Add other child lanes for other types of tests:
-            #  One lane for every collection containing only books
-            #   that are _not_ available.
-            #  A lane where all books belong to some series
-            #  A lane where all books use a particular DRM scheme/format
+                # Give each Worklist a name that is distinctive
+                # and easy for a client to parse.
+                if collection.data_source:
+                    data_source_name = collection.data_source.name
+                else:
+                    data_source_name = "[Unknown]"
+                display_name = "[Collection test] - License source {%s} - Medium {%s} - Collection name {%s}" % (data_source_name, medium, collection.name)
```

Comment:
The availability status is no longer mentioned in the lane name, because whoever requested the feed had a chance to specify a value for the `availability` facet. They ought to know what they requested, so there's no reason to spell t out in the feed itself.